global_costmap:
  global_frame: odom
  robot_base_frame: base_footprint

  update_frequency: 10.0
  publish_frequency: 10.0
  transform_tolerance: 0.5

  origin_x: 3.0
  origin_y: 1.0

  # static_map: false  
  static_map: true  
  rolling_window: false
  # rolling_window: true
  width: 6.0
  height: 6.0
  resolution: 0.05

  # plugins not used